packages/broker/src/broker.ts

```diff
@@ -56,7 +56,7 @@ export const createBroker = async (config: Config): Promise<Broker> => {
                 httpServer = await startHttpServer(httpServerRoutes, config.httpServer, apiAuthenticator)
             }
 
-            const nodeId = await streamrClient.getNodeId()
+            const nodeId = (await streamrClient.getNode()).getNodeId()
```

Comment:
Hmm I thought the intention is that we move more to calls being delegated to node via client?